groups/chips-rep-db/ec2.tf

```diff
@@ -46,6 +46,33 @@ module "db_ec2_security_group" {
   egress_rules = ["all-all"]
 }
 
+resource "aws_security_group_rule" "Oracle_Management_Agent" {
+ type = "ingress"
+ description = "Oracle Management Agent"
+ from_port = 3872
+ to_port = 3872
+ protocol = "tcp"
+ security_group_id = "${var.chips_rep_oem_sg}"
+}
+
+resource "aws_security_group_rule" "Enterprise_Manager_Upload_Http_SSL" {
+ type = "ingress"
+ description = "Enterprise Manager Upload Http SSL"
+ from_port = 4903
+ to_port = 4903
+ protocol = "tcp"
+ security_group_id = "${var.chips_rep_oem_sg}"
+}
+
+resource "aws_security_group_rule" "OEM_SSH" {
+ type = "ingress"
+ description = "OEM SSH"
+ from_port = 22
+ to_port = 2
+ protocol = "tcp"
+ security_group_id = "${var.chips_rep_oem_sg}"
+}
+
```

Comment:
The indenting needs a lot of attention here.

The `security_group_id` parameter expects the security group ID of the SG the rule will be attached to, so this should be `module.db_ec2_security_group.this_security_group_id`. This then needs to be coupled with a `source_security_group_id` parameter to define the source.

This parameter expects an ID, not a name, and so will need to be coupled with a data lookup to get the ID for the appropriate security group.

E.g.:
```
data "aws_security_group" "oem" {
  filter {
    name   = "tag:Name"
    values = [var.chips_rep_oem_sg]
  }
}
```

This can then be referenced via `data.aws_security_group.oem.id`.

E.g.:
```
resource "aws_security_group_rule" "Oracle_Management_Agent" {
  type                     = "ingress"
  description              = "Oracle Management Agent"
  from_port                = 3872
  to_port                  = 3872
  protocol                 = "tcp"
  source_security_group_id = data.aws_security_group.oem.id
  security_group_id        = module.db_ec2_security_group.this_security_group_id
}
```